cypress/e2e/tests/migration/applicationinventory/assessment/create.test.ts

```diff
@@ -102,6 +103,16 @@ describe(["@tier2"], "Application validations", () => {
         inputText(applicationNameInput, data.getFullName());
         cy.get(commonView.submitButton).should("not.be.disabled");
 
+        // Contributors Validation, Polarion TC 331
+        inputText(applicationContributorsInput, stakeHolders[0].name);
+        cy.get("button").contains(stakeHolders[0].name).click();
+        inputText(applicationContributorsInput, stakeHolders[1].name);
+        cy.get("button").contains(stakeHolders[1].name).click();
+        cy.get(applicationContributorsInput)
```

Comment:
Usually I'm trying to avoid such code in the test, consider to create a separate function or method that will reusable as much as possible and just to call it here, so it will be just a single line calling particular peace of code

Reply:
I understand your point, but this validation is something that will be used only here.

Later, I'll add the appropriate logic to the application class to fill this and owner values.